examples/chip-tool/commands/discover/Commands.h

```diff
@@ -22,57 +22,47 @@
 #include "DiscoverCommissionablesCommand.h"
 #include "DiscoverCommissionersCommand.h"
 #include <controller/DeviceAddressUpdateDelegate.h>
-#include <lib/dnssd/Resolver.h>
+#include <lib/address_resolve/AddressResolve.h>
 
-class Resolve : public DiscoverCommand, public chip::Dnssd::OperationalResolveDelegate
+class Resolve : public DiscoverCommand, public chip::AddressResolve::NodeListener
 {
 public:
-    Resolve(CredentialIssuerCommands * credsIssuerConfig) : DiscoverCommand("resolve", credsIssuerConfig) {}
+    Resolve(CredentialIssuerCommands * credsIssuerConfig) : DiscoverCommand("resolve", credsIssuerConfig)
+    {
+        mNodeLookupHandle.SetListener(this);
+    }
 
     /////////// DiscoverCommand Interface /////////
     CHIP_ERROR RunCommand(NodeId remoteId, uint64_t fabricId) override
     {
-        ReturnErrorOnFailure(mDNSResolver.Init(chip::DeviceLayer::UDPEndPointManager()));
-        mDNSResolver.SetOperationalDelegate(this);
-        ChipLogProgress(chipTool, "Dnssd: Searching for NodeId: %" PRIx64 " FabricId: %" PRIx64 " ...", remoteId, fabricId);
```

Comment:
Why did this logging get removed?

Reply:
I replaced the entire content of the command , so I ended up removing all existing content. Did not seem like a very important log as it basically repeats command-line arguments so it will likely not provide extra information for the user (although it could be used to validate that parsing is ok).

I can add it back if it seems important.